classes/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './items'
+
+class Music < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, publish_date, archived, genrename)
+    super(on_spotify, publish_date, archived, genrename)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super || @on_spotify
```

Comment:
- [ ] Consider for this method that should return true if parent's method returns true AND if on_spotify equals true not OR.

Reply:
This change has been done well